modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,35 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return array autoloaded data - autoload size, entries, option and its names.
+ */
+function perflab_aao_autoloaded_options() {
+	global $wpdb;
+	$option_threshold = apply_filters( 'perflab_aao_autoloaded_option_table_threshold', 100 );
```

Comment:
Can you please add a document for the filter? Something like below?

```
/**
 * Filters max bytes threshold to get options from Database.
 *
 * @since n.e.x.t
 *
 * @param int $limit Autoloaded options threshold size. Default 100.
 */
```